test/test_core.py

```diff
@@ -3761,16 +3761,22 @@ def test_dlfcn_asyncify(self):
   @requires_jspi
   @needs_dylink
   def test_dlfcn_jspi(self):
-    side_flags = ['-sSIDE_MODULE']
-    if self.get_setting('MEMORY64'):
-      side_flags.append('-sMEMORY64')
-    self.set_setting('JSPI')
-    self.run_process([
-      EMCC,
-      '-o', 'side.so',
-      test_file('core/test_dlfcn_jspi_side.c'),
-      *side_flags])
-    self.do_run_in_out_file_test('core/test_dlfcn_jspi.c', emcc_args=['side.so', '-sMAIN_MODULE=2'])
+      side_flags = []
+      if self.get_setting("MEMORY64"):
+          side_flags.append("-sMEMORY64")
+      self.run_process(
+          [
+              EMCC,
+              "-o",
+              "side.so",
+              test_file("core/test_dlfcn_jspi_side.c"),
+              "-sSIDE_MODULE",
+          ]
+          + self.get_emcc_args()
+      )
+      self.do_run_in_out_file_test(
+          "core/test_dlfcn_jspi.c", emcc_args=["side.so", "-sMAIN_MODULE=2"]
+      )
```

Comment:
We use 2 space indention.   We also allow tend to allow long line in test code so I think these can bot be just single lines.